reana_resources_k8s/templates/deployments/workers-template.yaml

```diff
@@ -24,10 +24,10 @@ spec:
         - name: EXPERIMENT
           value: {{worker.EXPERIMENT}}
         - name: SHARED_VOLUME
-          value: "/reana/{{worker.EXPERIMENT}}"
```

Comment:
It shouldn't affect because `SHARED_VOLUME_MOUNT_POINT` represents as a global variable the mount point for the shared volumes in every worker 🙅 . Therefore, since I didn't come straight away with the answer and because you had to ask, it is obvious that it is not clear so I think we should move the value for the mount point under the worker config: `worker.SHARED_VOLUME.MOUNT_POINT`